app/src/main/java/com/github/pockethub/android/ui/repo/RepositoryListAdapter.java

```diff
@@ -81,12 +84,16 @@ protected void updateDetails(final String description,
 
         if (!TextUtils.isEmpty(description)) {
             setText(1, description).setVisibility(View.VISIBLE);
+
         } else {
             setGone(1, true);
         }
 
         if (!TextUtils.isEmpty(language)) {
-            setText(2, language).setVisibility(View.VISIBLE);
+            GitHubColorUtils gitHubColorUtils = GitHubColorUtils.getInstance();
+            StyledText t = new StyledText();
+            t.foreground(language, Color.parseColor(gitHubColorUtils.githubColorCode(language)));
```

Comment:
Should we check whether gitHubColorUtils.githubColorCode(language) == null here? because some language's color is null in the githubcolor.json.